### ISSUE TITLE
Neat requiring activation after Windows 10 V21H2

### ISSUE THREAD

**Issue by mlatx** [kevingigiano/neat-legacy#3]
Hi Kevin,
  First of all, thank you for addressing the Neat Legacy sometime back.  I successfully used your code when Neat was forcing everyone to use their Cloud based service.

  I have now run into the same issue as before after upgrading to Windows 10 Version 21H2.  Unfortunately I had issues this summer with the upgrade and had to completely reinstall windows wiping out all my apps and only being able to save my user data.  I've attempted to use the same process as before but with no success.  I encounter that same forced logon to activate the software.  I did have an account long ago but it was never activated for the cloud. 

  Attached is my screenshot.

![Neat-screen shot](https://user-images.githubusercontent.com/113646619/190525965-743d0f36-962b-4360-a536-60832c64e7eb.PNG)

  Do you have any recommendations?

   In advance, thank you so very much!
Best Regards,
Mary Lou

**Comment by kevingigiano** [comment]
Hi Mary Lou,

Based on your screenshot, the patch I created is not installed.
Follow the instructions over at [https://github.com/kevingigiano/neat-legacy](https://github.com/kevingigiano/neat-legacy).

If you are still having trouble, send me an email at kevin.gigiano AT gmail.com and we can do a Zoom call or something.

Kevin

**Comment by mlatx** [comment]
Hi Kevin,
  Thank you so very much for pointing me in the correct direction.  I had tried installing the patch multiple times with no success.  This time I compared which files should have copied and found one of them was missing.  I attempted to overlay the one there with no success.  Had to stop the services first then was successful in getting the file copied over.  Just an fyi.. the file I had an issue with was the NeatCompany.NeatWorks.Logging.dll.

  Again, thank you so very much for your patches as we still use our scanner (and have for years).  I'm just wondering if I'll run into issues whenever we head to Windows 11 :)

  Have a wonderful day/weekend!
mla
